src/client/particles.cpp

```diff
@@ -186,10 +187,15 @@ video::SColor Particle::updateLight(ClientEnvironment *env)
 		light = blend_light(env->getDayNightRatio(), LIGHT_SUN, 0);
 
 	u8 m_light = decode_light(light + m_p.glow);
+
+	video::SColor light_color{0xFFFFFFFF};
+
+	final_color_blend(&light_color, m_light, env->getDayNightRatio(),
+			env->getLocalPlayer()->getLighting().ambient_light);
```

Comment:
I think no. `ContentCAO` has the similar code of the light applying. However, it gets the day and night banks and adds to them the glow separately, then blend them. Here the banks are blended at first, then the glow is added. I assume the results should be identical. I only deleted the decoding light because it returns the light in 0 - 255 range, not in 0 - LIGHT_SUN.